packages/typespec-test/test/openai_modular/generated/typespec-ts/src/api/operations.ts

```diff
@@ -4,6 +4,7 @@
 import {
   OpenAIContext as Client,
   isUnexpected,
+  ChatMessage,
```

Comment:
I may not understand why we have model's dependency from RLC not api layer.

Reply:
Because those imports are automactially fixed by ts-morph, it will "helpfully" add the first one it can find, I should move models/models.ts to the up.

Reply:
fixed